## Nanopore Data Visualisation

The idea is to build a collection of scripts that were used to create plots for (published) studies. So, it can benefit students and researchers in a way that they can reproduce figures using own data ([Example 1](https://github.com/pschloss/Kozich_ReAnalysis_AEM_2017/blob/master/code/plot_nmds.R) and [Example 2](https://github.com/bioinf/weswgs)). Let's make nanopore research more visual!

- **nanoPy** = for Python scripts
- **nanopoRe** = for scripts written in R

### Tools and Packages

1. [NanoPack: visualizing and processing long-read sequencing data](https://github.com/wdecoster/nanopack)
    - Figure 1 and 2, [Structural variants identified by Oxford Nanopore PromethION sequencing of the human genome](https://genome.cshlp.org/content/early/2019/06/11/gr.244939.118.abstract)
2. [Nano-snakemake](https://github.com/wdecoster/nano-snakemake)
3. [pycoQC - interactive QC plots](https://github.com/a-slide/pycoQC)
4. [poRe: an R package for the visualization and analysis of nanopore sequencing data](https://academic.oup.com/bioinformatics/article/31/1/114/2365693)
5. [SquiggleKit: A toolkit for manipulating nanopore signal data](https://github.com/Psy-Fer/SquiggleKit)
6. [Tombo (for analysis and visualization of raw nanopore signal)](https://github.com/nanoporetech/tombo)
7. [NanoSatellite](https://github.com/arnederoeck/NanoSatellite)
8. [Counterr](https://github.com/dayzerodx/counterr)
9. [NanoBlot: A Simple Tool for Visualization of RNA Isoform Usage From Third Generation RNA-sequencing Data](https://www.biorxiv.org/content/10.1101/2022.10.26.513894v1)

### Contribution

- You can contribute by forking and submitting pull requests, or posting suggestions as issues. 
- Please mention figure number (and name/doi of publication) in your script.
